index.js

```diff
@@ -2179,6 +2184,13 @@
     };
   }
 
+  //  see :: (String, { name :: String, url :: String }) -> String
+  function see(label, record) {
+    return record.url &&
```

Comment:
I'll freely admit that I'm not in the habit of considering how code will interoperate with old versions of related packages. Tricky!

Your fix sounds good to me. Would you like to submit a pull request?